sponnet/pipeline.libsonnet

```diff
@@ -19,7 +19,7 @@
     withTemplate(templateArtifact):: self + { template: templateArtifact },
     withSchema(schema):: self + { schema: schema },
     withInherit(inheritedFields):: self + if std.type(inheritedFields) == 'array' then { inherit: inheritedFields } else { inherit: [inheritedFields] },
-    withVariableValues(variables):: self + { variables: variables }, // variables are key-value pairs of <variable name> -> <variable value>
+    withVariableValues(variables):: self + { variables: variables },  // variables are key-value pairs of <variable name> -> <variable value>
```

Comment:
might just be late in the day for my tired eyes but is this just an extra space added by accident?

Reply:
Indeed, it's just the result of `jsonnet fmt` on the file.
I think it's good to run this over the library.